### ISSUE TITLE
wiki 

### ISSUE THREAD

**Issue by vondele** [glinscott/fishtest#1554]
Not sure where the best place is to mention this, but let me open an issue for wiki issues.

@vdbergh https://hardy.uhasselt.be/Fishtest/normalized_elo_practical.pdf is a dead link on https://github.com/glinscott/fishtest/wiki/Fishtest-mathematics

**Comment by ppigazzini** [comment]
It seems that the website is down:
https://hardy.uhasselt.be/

**Comment by vdbergh** [comment]
> It seems that the website is down: https://hardy.uhasselt.be/

Yes. I am reinstalling the OS. It goes a bit slow since I have to restore a backup from a very slow external hard drive.

**Comment by ppigazzini** [comment]
What OS?

**Comment by vdbergh** [comment]
It used to be CentOS 6. Now Ubuntu 22.04 LTS.

**Comment by ppigazzini** [comment]
Please mind that MongoDB Community support for Ubuntu 22.04 x86 is still a work in progress, see
https://www.mongodb.com/community/forums/t/installing-mongodb-over-ubuntu-22-04/159931/57
https://jira.mongodb.org/browse/SERVER-62300

**Comment by vdbergh** [comment]
For testing I am currently using an AWS instance with Ubuntu 22.04 (using the wiki install script for Fishtest). MongoDB seems to work fine (the only issue is that it sometimes doesn't start at boot, so I have to start it manually).

**Comment by ppigazzini** [comment]
I read in some posts that the new installer seems already on the MongoDB repo, but the documentation still shows only ARM support for Ubuntu 22.04. The fishtest server script checks for the Ubuntu version, so hopefully is downloading the correct (beta?) version.

```bash
ubuntu_release=$(lsb_release -c | awk '{print $2}')
echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu ${ubuntu_release}/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-6.0.list
```

**Comment by vdbergh** [comment]
The version is 6.0.3. That's all I know...

**Comment by vdbergh** [comment]
Interesting  `lsb_release` replies with  `No LSB modules are available`.

**Comment by vdbergh** [comment]
The website is up again.
